misago/users/models/deleteduser.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.utils import timezone
+
+
+DELETED_BY = (
+    (1, 'By User'),
+    (2, 'By Staff'),
+    (3, 'By System'),
+)
+
+
+class DeletedUser(models.Model):
+    deleted_on = models.DateTimeField(default=timezone.now)
+    deleted_by = models.PositiveIntegerField(
+        choices=DELETED_BY,
+        default=1
+    )
+    
+    class Meta:
+        get_latest_by = "deleted_on"
```

Comment:
Please use `ordering = ['deleted_on']` instead, and set `db_index=True` on `deleted_on` field.